test/functional/tests/install/test_install_rpm.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from time import sleep
+import pytest
+
+from api.cas import casadm, installer
+from core.test_run import TestRun
+
+install_command = 'rpm --install'
+uninstall_command = 'rpm --erase'
+all = '--allmatches'
+ignore = '--replacefiles'
+
+
+@pytest.mark.remote_only()
+@pytest.mark.without_install()
+def test_install_from_rpm():
+    """
+        title: Test for installing OpenCAS from RPM
+        description: |
+          Prepare SRPM for OpenCAS, build OpenCAS's RPM from SRPM,
+          install OpenCAS from RPM and then uninstall OpenCAS.
+        pass_criteria:
+          - OpenCAS installs successfully
+          - OpenCAS uninstalls successfully
+    """
+    with TestRun.step("Download and build OpenCAS."):
+        installer.download_opencas(5)
+        installer.build_opencas()
+
+    with TestRun.step("Create OpenCAS's SRPM."):
+        TestRun.executor.run_expect_success(f"make srpm -C {TestRun.usr.working_dir}")
+
+    with TestRun.step("Create OpenCAS's RPM."):
+        output = TestRun.executor.run_expect_success(
+            f"make rpm -C {TestRun.usr.working_dir}").stdout.splitlines()
+        interesting_output = []
+        for line in output:
+            if "RPMS/x86_64" in line:
+                interesting_output.append(line)
+            elif "You can find your fresh packages in" in line:
+                interesting_output.append(line)
+        program = interesting_output[0].split("/")[-1]
+        module = interesting_output[1].split("/")[-1]
+        path = interesting_output[2].split("'")[1]
+
+    with TestRun.step("Install OpenCAS from RPM."):
+        TestRun.executor.run(f"{install_command} {path}/{module} {ignore}")
+        TestRun.executor.run(f"{install_command} {path}/{program} {ignore}")
+
+    with TestRun.step("Check if OpenCAS is installed correctly."):
+        sleep(1)
+        try:
+            casadm.print_version()
```

Comment:
There should also be a check if CAS is installed from RPM and not from sources. You can do this by checking the exit status of `rpm -qa | grep open-cas-linux`